src/ESPEasy-Globals.h

```diff
@@ -168,7 +168,7 @@
 //#define PLUGIN_BUILD_DEV
 
 //add this if you want SD support (add 10k flash)
-//#define FEATURE_SD
+#define FEATURE_SD
```

Comment:
This should not be defined in the PR.
Or else it will be enabled for anyone.

The unintentional benefit for now is that Travis will test your build.
If Travis build is successful, will you add an extra commit to put this in comments again?

Reply:
The last commit already does that. Leaving these two unnecessary commits in the PR was not my intention. Sorry for that.